src/shared/components/oncoprint/TooltipUtils.ts

```diff
@@ -585,6 +585,15 @@ export function makeGeneticTrackTooltip(
                             `<img src="${customDriverImg}" title="${driver_filter}: ${driver_filter_annotation}" alt="driver filter" style="height:11px; width:11px;margin-left:3px"/>`
                         );
                     }
+                    //If we have data for the binary custom driver annotations, append an icon to the tooltip with the annotation information
+                    if (driver_filter && driver_filter === PUTATIVE_PASSENGER) {
```

Comment:
Small thing. This should be an `else if` since driver (previous condition) and passenger are mutually exclusive.

Reply:
Done